packages/api/cms-api/src/dam/files/dto/file.input.ts

```diff
@@ -89,6 +89,12 @@ export class CreateFileInput {
     // TODO is this validation even used?
     @IsObject()
     scope?: DamScopeInterface;
+
+    @IsOptional()
+    importSourceId?: string;
```

Comment:
We could adapt the validation to require an importSourceId if importSourceType is set